README.md

```diff
@@ -1643,11 +1648,29 @@ When running a job on the Config (Shared Filesystem) backend, Cromwell provides
         }
       }
 ```
+# Imports
+
+Import statements inside of a WDL file are supported by Cromwell when running in Server mode as well as Single Workflow Runner Mode.
+
+In Single Workflow Runner Mode, you pass in a zip file which includes the WDL files referenced by the import statements. Cromwell requires the zip file to be passed in as a command line argument, as explained by the section [run](#run)
+
+For example, given a workflow `wf.wdl` and an imports directory `WdlImports.zip`, a sample command would be:
+```
+java -jar cromwell.jar wf.wdl wf.inputs - - WdlImports.zip
+```
+
+In Server Mode, you pass in a zip file using the parameter `wdlDependencies` via the [POST /api/workflows/:version](#post-apiworkflowsversion) endpoint.
+
+
 # Sub Workflows
 
 WDL allows the execution of an entire workflow as a step in a larger workflow (see WDL SPEC for more details), which is what will be referred to as a sub workflow going forward.
 Cromwell supports execution of such workflows. Note that sub workflows can themselves contain sub workflows, etc... There is no limitation as to how deeply workflows can be nested.
 
+# Meta blocks
+
+In order to write notes
+
 ## Execution
```

Comment:
This Execution section as well as everything up to the REST API section are actually sub sections for sub workflows.
The Meta blocks section should be either above Sub Workflows or right before REST API